### DISCUSSION TITLE
Can we reply directly to an discussion from email instead of interacting to the dicussion via the web.

### DISCUSSION THREAD

**Discussion by bberkeyU** [NCAR/ucomp-pipeline#262] [General]
@mgalloy 
@bberkeyU 

Please reply via email with email or other artifacts to see how results show up here.

**Comment by mgalloy** [comment]
I am replying to this discussion via email.

-Mike

Michael Galloy
NSF NCAR/HAO — Mauna Loa Solar Observatory (MLSO)
Boulder, CO

**Reply by mgalloy** [reply]
I don't like how my reply has a bunch of other replies at the bottom, hidden by default, but still there.

**Comment by bberkeyU** [comment]
I have attached a fuzzy solar eclipse image with a lot of clouds a 10%
coverage.

**Reply by mgalloy** [reply]
I don't see the image.

**Comment by mgalloy** [comment]
I would recommend not replying via email, if at all possible. There are features that are not available, at least for Issues, when replying via email. Email replies also have the clutter of the each reply having all previous replies beneath them.

**Reply by bberkeyU** [reply]
Good point. I was just hoping that there was a way to give quick comments in email flow without jumping over to Git Hub.
